### PR TITLE
add the CodeStyle nuget package

### PR DESCRIPTION
**Pull request by axylophon**
Nuget package to distribute a common .editorconfig file between projects.

### CODE REVIEW

File: src/Adliance.Buddy.CodesStyle/Adliance.Buddy.CodesStyle.csproj [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <NoDefaultExcludes>true</NoDefaultExcludes><!-- include files starting with . -->
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
+  </PropertyGroup>
+  
+  <!-- Select the MSBuild props and composed .editorconfig files to pack -->
+  <ItemGroup>
+    <None Include="build\Adliance.Buddy.CodesStyle.props" Pack="true" PackagePath="build\" />
+    <None Include="Rules\.editorconfig" Pack="true" PackagePath="content\rules"></None>
+    <None Include="readme.md" Pack="true" PackagePath="\"/>
```
Comment (saxx) [inline src/Adliance.Buddy.CodesStyle/Adliance.Buddy.CodesStyle.csproj:23]: Why include the `readme` in the package?

**Comment by saxx** [comment]
Thanks! I'm not thrilled about the name `CodesStyle`. What do you think about `CodeStyle` or `CodeStyles`?

File: src/Adliance.Buddy.CodesStyle/Adliance.Buddy.CodesStyle.csproj (hunk 1 above)
Comment (axylophon) [reply src/Adliance.Buddy.CodesStyle/Adliance.Buddy.CodesStyle.csproj:23]: To render the readme on nuget.org:
https://devblogs.microsoft.com/nuget/add-a-readme-to-your-nuget-package/

Do you know a different way?

**Comment by axylophon** [comment]
> Thanks! I'm not thrilled about the name `CodesStyle`. What do you think about `CodeStyle` or `CodeStyles`?

Names are always hard ;) - do you have a different suggestion?

**Comment by saxx** [comment]
I'd prefer `CodeStyle` (without the hard to pronounce `s` in the middle).

**Comment by axylophon** [comment]
> I'd prefer `CodeStyle` (without the hard to pronounce `s` in the middle).

Now I see -> thats a typo and not desired ;)

File: src/Adliance.Buddy.CodesStyle/Adliance.Buddy.CodesStyle.csproj (hunk 1 above)
Comment (saxx) [reply src/Adliance.Buddy.CodesStyle/Adliance.Buddy.CodesStyle.csproj]: No ;) Did not think about that, thanks.

**Pull request merged**